### ISSUE TITLE
Issue with ignored files

### ISSUE THREAD

**Issue by soumak77** [brunch/eslint-brunch#16]
If you have a file `app/game/index.js` and a `.eslintignore` file with contents:

```
app/game/*
```

then run brunch, you'll get the following error:

```
error: Linting of app/game/index.js failed. Cannot read property 'errorCount' of undefined 
```

This is caused by the following line in `eslint-brunch`:

```
this.linter.executeOnText(data, path).results[0]
```

The problem is that the results array is empty if the file is ignored by eslint, but matched by eslint-brunch, therefore `results[0]` is undefined.

**Comment by soumak77** [comment]
The code refactoring done by #17 may solve this issue.  Can you publish a new version with that fix?

**Comment by spyl94** [comment]
Sure, I'm going to test and release a new version if it fix the issue

**Comment by spyl94** [comment]
Just released `eslint-brunch@3.12.0`! Thanks 👍
